open-hackathon-client/src/client/templates/home.html

```diff
@@ -151,7 +151,7 @@
                                                 </div>
                                                 <div>
                                                     <span class="fa fa-map-marker"></span>
-                                                    {{ item.location or "线上" | limitTo(2) }}
+                                                    {{ item.location[0:10] or "线上" | limitTo(2) }}
```

Comment:
What's `limitTo` for? The name telling me it might be used to limit the length too. Can you investigate it more or ask Boli for help. If it is, we don't want two different ways to do string slice